Styling/README.md

```diff
@@ -622,6 +624,31 @@ Converts the number from radians to degrees.
 }
 ```
 
+#### clamp
+
+`clamp(value : Number,  min : Number, max : Number) : Number`
+
+Returns value if it is greater than min and less than max. Otherwise, if value is less than min, min is returned, and if value is greater than max, max is returned.
```

Comment:
This description could be more concise. Cesium's description is pretty good:

`Constrains a value to lie between two values.`